recipes/libproperties/all/test_package/example.cpp

```diff
@@ -0,0 +1,6 @@
+#include <iostream>
```

Comment:
Please use an example testing the libproperties library.
So using `properties_source_file_read`, `properties_parse` and `properties_source_string_read`.
You can add a sample properties file to parse.

Also, make the test pure C, so no cpp file.